README.md

```diff
@@ -28,15 +28,15 @@ What is the purpose of the `alt` attribute? e.g:
 ```
 
 ```text
-Your answer here.
+Should your image not work, or be down, it provides context for what should be there. Also for accessibility by the americans with disabilities act. 
+### Question #3
 ```
 
-### Question #3
 
 What is the purpose of the `<head></head>` tag in HTML?
 
 ```text
-Your answer here.
+the <head></head> tag dictates what information is displayed in the top of a webpage. It is not visible within the actual html document, only in the description part at the top of the page. 
```

Comment:
I think you're referring to the `<title>` tag here, but the `<head>` does more than just determine what shows up at the top of the page. It contains rules, meta-data and other information that affects the web page but does not necessarily show up on the page.